docs/kerberos.md

```diff
@@ -170,7 +171,7 @@ Submit the job with the keytab:
 
     dcos spark run --submit-args="\
     --kerberos-principal user@REALM \
-    --keytab-secret-path /__dcos_base64__hdfs-keytab \
+    --keytab-secret-path /spark/__dcos_base64__hdfs-keytab \
```

Comment:
If I understand correctly, `/spark` **must** be the dispatcher ID now, right? Would it make sense to document this as well?